roles/v2/mining-proxy/src/lib/job_negotiator/mod.rs

```diff
@@ -91,17 +103,19 @@ impl JobNegotiator {
                     );
                 match next_message_to_send {
                     Ok(SendTo::None(Some(JobNegotiation::SetCoinbase(m)))) => {
-                        let sender = self_mutex
-                            .safe_lock(|s| s.sender_coinbase_output_max_additional_size.clone())
-                            .unwrap();
                         let coinbase_output_max_additional_size = CoinbaseOutputDataSize {
                             coinbase_output_max_additional_size: m
                                 .coinbase_output_max_additional_size,
                         };
-                        sender
+                        let out_script = self_mutex
+                            .safe_lock(|s| s.last_coinbase_out.clone())
+                            .unwrap()
+                            .unwrap();
```

Comment:
Is this (second) `unwrap` safe? If so, I'm curious, what makes it safe (ie. what guarantees that `last_coinbase_out` is not `None`)?